src/NotesHApp.js

```diff
@@ -7,25 +8,25 @@ import REMOVE_NOTE_MUTATION from './graphql/removeNoteMutation'
 
 import './NotesHApp.css'
 
-function NotesHApp() {
-  const { data: { listNotes } = { listNotes: [] } } = useQuery(LIST_NOTES_QUERY)
+function NotesHApp () {
+  const { data: { listNotes } = { listNotes: [] } } = useQuery(LIST_NOTES_QUERY, { pollInterval: 500 })
```

Comment:
we don't need to poll the notes list. As the user is the only one who ever updates it, we can just refetch queries on create.